RELEASE_NOTES.md

```diff
@@ -56,7 +56,7 @@ This is built on the Jupyter Notebook v6.5.6 and IPython 8.25.0 (more notes will
   - scikit-learn: 1.2.1 -> 1.5.0
   - statsmodels: 0.14.2 (new)
   - terminado: 0.17.1 -> 0.18.0
-  - tornado: 6.2 -> 6.4
+  - tornado: 6.2 -> 6.4.1
```

Comment:
Of course there's another update just when I thought I had all the python stuff done...

Reply:
😬